compoments/report-repair/existing-repair.js

```diff
@@ -0,0 +1,41 @@
+import PropTypes from 'prop-types';
+import RadioFieldSet from '../radioFieldSet';
+import React from 'react';
+
+const ExistingRepair = ({ handleChange, values }) => {
+  const name = 'existing-repair';
+  const title =
+    'Is this a new repair request or have you reported this before?';
+  const options = [
+    { value: 'yes', title: 'New repair' },
```

Comment:
Logic: could be difficult for future developers to understand. The name of the form is "existing repair" and yet the yes choice is "new repair" - could we change it to "new" & "existing" rather than "yes" and "no"?